language/python/develop.md

```diff
@@ -4,8 +4,253 @@ keywords: python, local, development, run,
+
+## Local database and containers
+
+First, we’ll take a look at running a database in a container and how we use volumes and networking to persist our data and allow our application to talk with the database. Then we’ll pull everything together into a Compose file which allows us to setup and run a local development environment with one command. Finally, we’ll take a look at connecting a debugger to our application running inside a container.
+
+Instead of downloading MySQL, installing, configuring, and then running the MySQL database as a service, we can use the Docker Official Image for MySQL and run it in a container.
+
+Before we run MySQL in a container, we'll create a couple of volumes that Docker can manage to store our persistent data and configuration. Let’s use the managed volumes feature that Docker provides instead of using bind mounts. You can read all about [Using volumes](../../storage/volumes.md) in our documentation.
+
+Let’s create our volumes now. We’ll create one for the data and one for configuration of MongoDB.
+
+```shell
+$ docker volume create mysql
+$ docker volume create mysql_config
+```
+
+Now we’ll create a network that our application and database will use to talk to each other. The network is called a user-defined bridge network and gives us a nice DNS lookup service which we can use when creating our connection string.
+
+```shell
+$ docker network create mysqlnet
+```
+
+Now we can run MySQL in a container and attach to the volumes and network we created above. Docker pulls the image from Hub and run it for you locally.
+
+```shell
+$ docker run -it --rm -d -v mysql:/var/lib/mysql \
+  -v mysql_config:/etc/mysql -p 3306:3306 \
+  --network mysqlnet \
+  --name mysqldb \
+  -e MYSQL_ALLOW_EMPTY_PASSWORD=true \
+  mysql
+```
+
+Now, let’s make sure that our MySQL database is running and that we can connect to it. Connect to the running MySQL database inside the container using the following command:
+
+```shell
+$ docker run -it --network mysqlnet --rm mysql mysql -hmysqldb
+Enter password: ********
+
+Welcome to the MySQL monitor.  Commands end with ; or \g.
+Your MySQL connection id is 8
+Server version: 8.0.23 MySQL Community Server - GPL
+
+Copyright (c) 2000, 2021, Oracle and/or its affiliates.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+mysql>
+```
+
+In the above command, we used the same MySQL image to connect to the database but this time, we passed the ‘mysql’ command to the container with the `-h` flag containing the name of our MySQL container name. Press CTRL-D to exit the MySQL  interactive terminal.
+
+Okay, now that we have a running MySQL, let’s update `app.py` to use MySQL as a datastore. Let’s also add some routes to our server. One for fetching records and one for inserting records.
```

Comment:
Should we add here a link to the app setup in the `Build images` page as a reminder? Just in case we land directly on this page or forgot our application setup :sweat_smile: 
I would find useful also to have the directory structure/tree for the python app as we have to edit the files next:
```
python-docker
|____ app.py
|____ requirements.txt
|____ Dockerfile
```